File: realtor/html/js/shellData.js
function buildShellData(data) {
    shellData.apartment = []
    shellData.garage = []

    for (var i=0; i < data.shell.apartment.length; i++)
    {
        shellData.apartment.push({
            name : data.shell.apartment[i].name,
            key :  data.shell.apartment[i].key,
            shellType : data.shell.apartment[i].shellType,
            type : "shell"
        })
    }

    for (var i=0; i < data.shell.garage.length; i++)
    {
        shellData.garage.push({
            name : data.shell.garage[i].name,
            key :  data.shell.garage[i].key,
            shellType : data.shell.garage[i].shellType,
            type : "shell"
        })
    }

    if (data.smShell)
    {
        for (var i=0; i < data.smShell.length; i++)
            {
                var newData = {
                    name : data.smShell[i].name,
                    key :  data.smShell[i].key,
                    type : "smShell"
                }
        
                shellData.apartment.push(newData)
                shellData.garage.push(newData)
            }
    }
    
    if (data.smShellTemplate)
    {
        for (var i=0; i < data.smShellTemplate.length; i++)
        {
            var newData = {
                name : data.smShellTemplate[i].name,
                key :  data.smShellTemplate[i].key,
                type : "smShellTemplate"
            }
    
            shellData.apartment.push(newData)
            shellData.garage.push(newData)
        }
    }
}

File: configs/shellData.lua
RealtorShellData = {apartment={}, garage={}}
ShellData = {
    apartment = {
        ["shell_lester"] = {
            name = "Small Apartment",
            shellType = "shell", 
            offset = vec3(0, 0, 2),
            door = vec4(-1.60, -5.36, -0.37, 0)
        },
        ["aparment_1"] = {
            name = "Luxury Apartment",
            shellType = "mlo", 
            door = vec4(-781.747, 318.378, 217.673, 354.336)
        },
    },
    
    garage = {
        ["shell_garagem"] = {
            name = "Medium garage",
            shellType = "shell",
            offset = vec3(0, 0, 2),
            garageDoor = vec4(2.81, -4.10, -0.75, 0),
        },
        ["big_garage"] = {
            name = "Luxury Garage",
            shellType = "mlo", 
            garageDoor = vec4(231.530, -1003.248, -99.000, 351.333),
        }
    }
}


-- init --
for k,v in pairs(ShellData.apartment) do table.insert(RealtorShellData.apartment, {key = k, name = v.name, shellType = v.shellType}) end
for k,v in pairs(ShellData.garage) do table.insert(RealtorShellData.garage, {key = k, name = v.name, shellType = v.shellType}) end